hilti/include/rt/global-state.h

```diff
@@ -44,6 +46,9 @@ struct GlobalState {
     /** If not zero, `Configuration::abort_on_exception` is disabled. */
     int disable_abort_on_exceptions = 0;
 
+    /** getrusage() at library initialization time. */
+    struct rusage rusage_init;
```

Comment:
I'd much more prefer if we used a `ResourceUsage` here, if only to ensure that we can consistently collect and compare data we care about, e.g., in `hilti::rt::init` we could just have
```.cpp
rusage_init = rt::resource_usage();
```

Reply:
good idea